src/Components/DigitBoard.tsx

```diff
@@ -0,0 +1,22 @@
+import DigitButton from './DigitButton';
+
+const digits = ["9", "8", "7", "6", "5", "4", "3", "2", "1", "0"] as const;
```

Comment:
```suggestion
const digits = Array(10).fill(null).map((_,index) => index).reverse()
```

요런식으로 메서드체이닝으로 구해볼 수도 있어요